src/Collection/Stream/Collectors.php

```diff
@@ -14,25 +14,21 @@
 final class Collectors
 {
     /**
-     * @template T
```

Comment:
phpstan is correct:
```
  31     Template type T of method Munus\Collection\Stream\Collectors::toSet() is not referenced in a parameter.    
```

since is not referenced, returned collector first template param is treated as mixed anyway